doc/article/en-US/contact-manager-tutorial.md

```diff
@@ -329,7 +329,7 @@ Ok, now that we've got the contact list built, we need to use it. To do that, up
     <template>
       <require from="bootstrap/css/bootstrap.css"></require>
       <require from="./styles.css"></require>
-      <require from="./contact-list"></require>
```

Comment:
Check https://github.com/aurelia/app-contacts/blob/master/src/contact-list.js File name is correct.